src/content/docs/it/recipes.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Altre soluzioni
+description: Esempi guidati su come aggiungere nuove funzionalità al tuo progetto Astro.
+i18nReady: true
+---
+
+import RecipesNav from '~/components/RecipesNav.astro';
+
+Esempi guidati su come aggiungere nuove funzionalità al tuo progetto Astro.
+
+## Soluzioni ufficiali
+
+<RecipesNav />
+
+## Soluzioni dalla community
+
+Se hai una soluzione che vuoi condividere, consulta [le nostre linee guida per contribuire](https://github.com/withastro/docs/blob/main/contributor-guides/submitting-a-recipe.md)  
+
+- [Importare immagini utilizzando un nome file dinamico](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
+- [Aggiungere transizioni di pagina animate con Swup](https://navillus.dev/blog/astro-plus-swup)
+- [Utilizzare UnoCSS in Astro](https://www.elian.codes/blog/23-02-11-implementing-unocss-in-astro/)
+- [Costruire una tabella dei contenuti utilizzando le intestazioni dei file markdown di Astro](https://kld.dev/building-table-of-contents/)
+- [Creare un plugin Remark per rimuovere i "righini" dai tuoi file Markdown](https://eatmon.co/blog/remove-runts-markdown/)
+- [Aggiungere al tuo sito la funzionalità di ricerca con Pagefind](https://blog.otterlord.dev/post/astro-search/)
+- [Configurare VSCode, ESLint e Prettier in progetti Astro](https://patheticgeek.dev/blog/astro-prettier-eslint-vscode)
+- [Ottimizzare le build di progetti Astro: modificare i file HTML post-build](https://straffesites.com/en/blog/optimize-astro-html-post-build)
+- [Aggiungere la modalità scura in Astro con Tailwind CSS](https://www.kevinzunigacuellar.com/blog/dark-mode-in-astro/)
+- [Generare sitemaps localizzate nei tuoi progetti Astro con Storyblok](https://straffesites.com/en/blog/localized-sitemap-astro-storyblok)
+- [Integrare Prettier con Astro e Tailwind CSS](https://straffesites.com/en/blog/integrate-prettier-astro-tailwindcss)
+- [Aggiungere un feed RSS ad Astro utilizzando le storie di Storyblok](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
+- [Utilizzare tRPC in Astro](https://www.thomasledoux.be/blog/using-trpc-astro-islands-react)
+- [Aggiungere il supporto multilingua al tuo sito Astro con Crowdin: una guida alla localizzazione in Astro](https://crowdin.com/blog/2023/06/21/astro-localization-and-i18n)
+- [Aggiungere le immagini di un blog post al feed RSS in Astro](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
```

Comment:
```suggestion
- [Aggiungere le immagini di un blog post al feed RSS Astro](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
```

Astro should be treated as a brand and therefore we shouldn't add possessive prepositions to it unless they're present in the original document. It's the same as in "pasta della Barilla vs pasta Barilla".